src/bin/flamegraph.rs

```diff
@@ -75,14 +79,15 @@ fn main() {
     let flamegraph_filename: PathBuf = opt
         .output
         .take()
-        .unwrap_or("flamegraph.svg".into());
+        .unwrap_or_else(|| "flamegraph.svg".into());
```

Comment:
why this change?

Reply:
Note that this is in a separate commit, with the commit message noting it cleans up some clippy warnings. (clippy warns against the before pattern because it performs a potentially expensive operation regardless of whether it's needed or not.)